src/Schema/Events/FilterAndSort.ts

```diff
@@ -75,10 +75,41 @@ export interface AuctionResultsFilterParams {
  */
 export interface CommercialFilterParamsChanged {
   action: ActionType.commercialFilterParamsChanged
-  context_module: ContextModule.artworkGrid | ContextModule.auctionResults
+  context_module: ContextModule.artworkGrid
   context_owner_type: OwnerType
   context_owner_id?: string
   context_owner_slug?: string
-  current: CommercialFilterParams | AuctionResultsFilterParams
-  changed: CommercialFilterParams | AuctionResultsFilterParams
+  current: CommercialFilterParams
+  changed: CommercialFilterParams
+}
+
+/**
+ * A user applies filters to a filterable/sortable auction results module
+ *
+ * This schema describes events sent to Segment from [[auctionResultsFilterParamsChanged]]
+ *
+ *  @example
+ *  ```
+ *  {
+ *    action: "auctionResultsFilterParamsChanged",
+ *    context_module: "artistAuctionResults",
+ *    context_owner_type: "artist",
+ *    context_owner_id: "510ade6c4926534fd80000e9",
+ *    context_owner_slug: "alex-da-corte",
+ *    changed: {createdAfterYear: 2014},
+ *    current: {createdAfterYear: 2008; sort: "DATE_DESC"}
+ *  }
+ * ```
+ */
+export interface AuctionResultsFilterParamsChanged {
+  action: ActionType.auctionResultsFilterParamsChanged
+  context_module:
+    | ContextModule.auctionResults
```

Comment:
This doesn't exist yet right? We're just preparing for when we create a new auction results page?

Reply:
yes. right now, this context module does not exist, but will serve as context for the auction results database